docs/community/20230104-mixed-clusters.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Feature Group for Mixed Clusters in Cluster API
+authors:
+  - "@prometherion"
+  - "@lentzi90"
+reviewers:
+  - "@lentzi90"
+  - "@richardcase"
+creation-date: 2023-01-04
+last-updated: 2023-01-06
+status: proposed
+see-also:
+  - https://github.com/kubernetes-sigs/cluster-api/issues/7475#issuecomment-1302454745
+  - https://kubernetes.slack.com/archives/C8TSNPY4T/p1667458956589639
+---
+# Mixed Clusters in Cluster API Feature Group
+
+This document briefly outlines the scope, communication media, and stakeholders for a formal Feature Group dedicated to defining a Cluster API-approved solution for "mixed clusters" which combines two or more infrastructure providers.
+
+## User Story and Problem Statement
+
+As a Kubernetes cluster administrator, I want the ability to specify different infrastructure providers for the Control Plane and the Worker nodes to decouple each-other infrastructure providers.
+
+At the current state, Cluster API is giving for granted that worker nodes and control plane components are backed by the same infrastructure provider. However, this is a hard statement, since the two components could be backed by different infrastructure providers.
+
+A practical example is offered with [Kamaji](https://github.com/clastix/kamaji) that offers virtualized Control Plane resources used by worker nodes provisioned on any cloud provider by using the `kubeadm` bootstrapper.
+
+## Scope
+
+The scope of this group will be the following:
+
+1. Creating a PoC demonstrating that a mixed cluster solution is possible
```

Comment:
Personally, I don't think this is required, it is possible as of today if the providers support this use case.
What about refocusing the POC on a show case of the API proposed below? 

Please also note that it is always a risk prototyping something not yet discussed with the community, but this is up to the working group members

Reply:
I agree. It has already been shown that it is possible to do it (although with various degrees of hacking and/or limitations).